mmcv/fileio/file_client.py

```diff
@@ -292,19 +523,44 @@ def get_text(self, filepath):
                 Defaults to None.
             force (bool, optional): Whether to override the backend if the name
                 has already been registered. Defaults to False.
+            prefixes (str or list[str] or tuple[str]): The prefix of the
+                registered storage backend.
+
+        .. versionadd:: 1.3.14
```

Comment:
It seems that this part does not show in https://mmcv--1330.org.readthedocs.build/en/1330/api.html#mmcv.fileio.FileClient.register_backend, did I miss something?

Reply:
The label `.. versionadd:: 1.3.14` does not work well and I will refactor it